Zenoh 1.8.0 causes tests to hang on Windows

### Generated by Generative AI

No

### Operating System:

Windows 10

### ROS version or commit hash:

rolling

### RMW implementation (if applicable):

_No response_

### RMW Configuration (if applicable):

_No response_

### Client library (if applicable):

_No response_

### 'ros2 doctor --report' output

<details><summary>ros2 doctor --report</summary>

```console
<COPY OUTPUT HERE>
```
</details>


### Steps to reproduce issue

Run a windows job on ci.ros2.org

### Expected behavior

Windows tests would pass

### Actual behavior

Windows tests hang and time out

```
        Start 131: test_qos_event__rmw_zenoh_cpp

131: Test command: C:\pixi_ws\.pixi\envs\default\python.exe "-u" "C:/ci/ws/install/share/ament_cmake_ros/cmake/run_test_isolated.py" "C:/ci/ws/build/rclcpp/test_results/rclcpp/test_qos_event.gtest.xml" "--package-name" "rclcpp" "--output-file" "C:/ci/ws/build/rclcpp/ament_cmake_gmock/test_qos_event__rmw_zenoh_cpp.txt" "--env" "RMW_IMPLEMENTATION=rmw_zenoh_cpp" "--command" "C:/ci/ws/build/rclcpp/test/rclcpp/Release/test_qos_event.exe" "--gtest_output=xml:C:/ci/ws/build/rclcpp/test_results/rclcpp/test_qos_event.gtest.xml"
131: Working Directory: C:/ci/ws/build/rclcpp/test/rclcpp
131: Test timeout computed to be: 60
131: -- run_test.py: extra environment variables:
131:  - RMW_IMPLEMENTATION=rmw_zenoh_cpp
131: -- run_test.py: invoking following command in 'C:\ci\ws\build\rclcpp\test\rclcpp':
131:  - C:\ci\ws\install\lib\rmw_test_fixture_implementation\run_rmw_isolated C:/ci/ws/build/rclcpp/test/rclcpp/Release/test_qos_event.exe --gtest_output=xml:C:/ci/ws/build/rclcpp/test_results/rclcpp/test_qos_event.gtest.xml
131: Running main() from gmock_main.cc
131: [==========] Running 12 tests from 1 test suite.
131: [----------] Global test environment set-up.
131: [----------] 12 tests from TestQosEvent
131: [ RUN      ] TestQosEvent.test_publisher_constructor
131: [       OK ] TestQosEvent.test_publisher_constructor (35 ms)
131: [ RUN      ] TestQosEvent.test_subscription_constructor
131: [2m2026-03-31T17:57:56.655843Z[0m [31mERROR[0m net-0 ThreadId(02) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:57:56.659598Z[0m [31mERROR[0m net-1 ThreadId(06) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:57:56.661305Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:57:56.661664Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_subscription_constructor (27 ms)
131: [ RUN      ] TestQosEvent.test_default_incompatible_qos_callbacks
131: [2m2026-03-31T17:57:56.681015Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:57:56.683438Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:57:56.685487Z[0m [31mERROR[0m acc-1 ThreadId(11) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:57:56.685795Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_default_incompatible_qos_callbacks (10026 ms)
131: [ RUN      ] TestQosEvent.construct_destruct_rcl_error
131: [2m2026-03-31T17:58:06.708389Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.708626Z[0m [33m WARN[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to find transport for link event: 5ef93dd056c70416de05f75d12ba9e6c
131: [2m2026-03-31T17:58:06.708814Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.709235Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [ERROR] [1774979886.715258600] [rclcpp]: Error in destruction of rcl event handle: error not set
131: [       OK ] TestQosEvent.construct_destruct_rcl_error (25 ms)
131: [ RUN      ] TestQosEvent.execute
131: [2m2026-03-31T17:58:06.732991Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.735282Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.737179Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.739132Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: C:\ci\ws\src\ros2\rclcpp\rclcpp\test\rclcpp\test_qos_event.cpp(281): Skipped
131: 
131: 
131: [  SKIPPED ] TestQosEvent.execute (26 ms)
131: [ RUN      ] TestQosEvent.add_to_wait_set
131: [2m2026-03-31T17:58:06.759769Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.763023Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.764902Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.765163Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.add_to_wait_set (26 ms)
131: [ RUN      ] TestQosEvent.test_on_new_event_callback
131: [2m2026-03-31T17:58:06.791749Z[0m [31mERROR[0m net-2 ThreadId(13) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.793372Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.793700Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: C:\ci\ws\src\ros2\rclcpp\rclcpp\test\rclcpp\test_qos_event.cpp(344): Skipped
131: 
131: 
131: [  SKIPPED ] TestQosEvent.test_on_new_event_callback (26 ms)
131: [ RUN      ] TestQosEvent.test_invalid_on_new_event_callback
131: [2m2026-03-31T17:58:06.813068Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.815281Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.817478Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.817638Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [WARN] [1774979886.819958100] [rclcpp]: Calling set_on_new_qos_event_callback for non registered publisher event_type
131: [WARN] [1774979886.819992700] [rclcpp]: Calling clear_on_new_qos_event_callback for non registered event_type
131: [WARN] [1774979886.820007100] [rclcpp]: Calling set_on_new_qos_event_callback for non registered subscription event_type
131: [WARN] [1774979886.820015600] [rclcpp]: Calling clear_on_new_qos_event_callback for non registered event_type
131: [       OK ] TestQosEvent.test_invalid_on_new_event_callback (25 ms)
131: [ RUN      ] TestQosEvent.test_pub_matched_event_by_set_event_callback
131: [2m2026-03-31T17:58:06.838880Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.842209Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.842371Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.842735Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_pub_matched_event_by_set_event_callback (25 ms)
131: [ RUN      ] TestQosEvent.test_sub_matched_event_by_set_event_callback
131: [2m2026-03-31T17:58:06.864780Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.868107Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.869882Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.870186Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_sub_matched_event_by_set_event_callback (27 ms)
131: [ RUN      ] TestQosEvent.test_pub_matched_event_by_option_event_callback
131: [2m2026-03-31T17:58:06.893378Z[0m [31mERROR[0m net-1 ThreadId(06) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.896702Z[0m [31mERROR[0m net-2 ThreadId(13) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.898617Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.899302Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_pub_matched_event_by_option_event_callback (26 ms)
131: [ RUN      ] TestQosEvent.test_sub_matched_event_by_option_event_callback
131: [2m2026-03-31T17:58:06.917789Z[0m [31mERROR[0m acc-0 ThreadId(04) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.920183Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [2m2026-03-31T17:58:06.922135Z[0m [31mERROR[0m acc-2 ThreadId(15) [2mzenoh::api::admin[0m[2m:[0m Unable to publish link event: session closed
131: [2m2026-03-31T17:58:06.924061Z[0m [31mERROR[0m ThreadId(10) [2mzenoh::api::admin[0m[2m:[0m Unable to publish transport event: session closed
131: [       OK ] TestQosEvent.test_sub_matched_event_by_option_event_callback (26 ms)
131: [----------] 12 tests from TestQosEvent (10326 ms total)
131: 
131: [----------] Global test environment tear-down
131: [==========] 12 tests from 1 test suite ran. (10326 ms total)
131: [  PASSED  ] 10 tests.
131: [  SKIPPED ] 2 tests, listed below:
131: [  SKIPPED ] TestQosEvent.execute
131: [  SKIPPED ] TestQosEvent.test_on_new_event_callback
131/134 Test #131: test_qos_event__rmw_zenoh_cpp ..............................***Timeout  60.00 sec
```

### Additional information

Looks like the CI run caught it (138 new test failures on Windows), but it was merged anyways. Probably because the other 3 platforms were yellow with a different unrelated test failure.

https://github.com/ros2/rmw_zenoh/pull/935#issuecomment-4160970638

--- a/zenoh_cpp_vendor/CMakeLists.txt
+++ b/zenoh_cpp_vendor/CMakeLists.txt
@@ -32,12 +32,14 @@ else()
   # See https://github.com/eclipse-zenoh/zenoh-c/blob/c1a477d3f5efc3abc0f0cf95e0773225d00dea3b/Cargo.lock#L4229
   # pointing to zenoh https://github.com/eclipse-zenoh/zenoh/commit/69b410e705e94531902a41a51b74d0c7b098264c
   set(zenoh_c_commit c1a477d3f5efc3abc0f0cf95e0773225d00dea3b)
+  set(zenoh_c_patches patches-c1a477d3)
 
   # If Rust version below Zenoh MSRV (currently 1.88), use equivalent commit from branch ROS/rust-1.75
   # See https://github.com/eclipse-zenoh/zenoh-c/blob/0e3ec084142382096d09f7fc2ad59385998b92e6/Cargo.lock#L4237
   # pointing to zenoh https://github.com/eclipse-zenoh/zenoh/commit/f27282e78c6e42e04249349049fa68cfbc941369
   if(CARGO_VERSION VERSION_GREATER_EQUAL "1.75" AND CARGO_VERSION VERSION_LESS "1.88")
     set(zenoh_c_commit 0e3ec084142382096d09f7fc2ad59385998b92e6)
+    set(zenoh_c_patches patches-0e3ec084)
   endif()
 
   ament_vendor(zenoh_c_vendor
@@ -47,6 +49,7 @@ else()
       "-DZENOHC_CARGO_FLAGS=${ZENOHC_CARGO_FLAGS}"
       "-DZENOHC_BUILD_WITH_UNSTABLE_API=TRUE"
       "-DZENOHC_CUSTOM_TARGET=${ZENOHC_CUSTOM_TARGET}"
+    PATCHES "${zenoh_c_patches}"
   )
 
   ament_vendor(zenoh_cpp_vendor

--- a/zenoh_cpp_vendor/patches-0e3ec084/0001-zenoh-e5db0ce-from-0e3ec084.patch
+++ b/zenoh_cpp_vendor/patches-0e3ec084/0001-zenoh-e5db0ce-from-0e3ec084.patch
@@ -1,0 +1,292 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index dbec438e..d834231b 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -4234,7 +4234,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+  "arc-swap",
+@@ -4286,7 +4286,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-buffers"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "zenoh-collections",
+ ]
+@@ -4325,7 +4325,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-codec"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "tracing",
+  "uhlc",
+@@ -4337,7 +4337,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-collections"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+ ]
+@@ -4345,7 +4345,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-config"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "json5",
+  "nonempty-collections",
+@@ -4370,7 +4370,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-core"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "lazy_static",
+  "tokio",
+@@ -4381,7 +4381,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-crypto"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "aes",
+  "hmac",
+@@ -4394,7 +4394,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-ext"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "bincode",
+@@ -4413,7 +4413,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-keyexpr"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "getrandom 0.2.17",
+  "hashbrown 0.16.1",
+@@ -4428,7 +4428,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "zenoh-config",
+  "zenoh-link-commons",
+@@ -4448,7 +4448,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-commons"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4481,7 +4481,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-quic"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4506,7 +4506,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-serial"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "tokio",
+@@ -4524,7 +4524,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-tcp"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "socket2 0.5.10",
+@@ -4541,7 +4541,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-tls"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4570,7 +4570,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-udp"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "libc",
+@@ -4591,7 +4591,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-unixpipe"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "advisory-lock",
+  "async-trait",
+@@ -4613,7 +4613,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-unixsock_stream"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "nix 0.29.0",
+@@ -4631,7 +4631,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-vsock"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "libc",
+@@ -4649,7 +4649,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-ws"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "futures-util",
+@@ -4669,7 +4669,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-macros"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+@@ -4680,7 +4680,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-pinned-deps-1-75"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-lock",
+  "base64ct",
+@@ -4725,7 +4725,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-plugin-trait"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "git-version",
+  "libloading",
+@@ -4742,7 +4742,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-protocol"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "const_format",
+  "rand 0.8.5",
+@@ -4757,7 +4757,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-result"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "anyhow",
+ ]
+@@ -4765,7 +4765,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-runtime"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "lazy_static",
+  "ron",
+@@ -4779,7 +4779,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-shm"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "advisory-lock",
+  "async-trait",
+@@ -4808,7 +4808,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-stats"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+  "prometheus-client",
+@@ -4821,7 +4821,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-sync"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "arc-swap",
+  "event-listener",
+@@ -4835,7 +4835,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-task"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "futures",
+  "tokio",
+@@ -4848,7 +4848,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-transport"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "crossbeam-utils",
+@@ -4883,7 +4883,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-util"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=f27282e78c6e42e04249349049fa68cfbc941369#f27282e78c6e42e04249349049fa68cfbc941369"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?rev=e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "const_format",

--- a/zenoh_cpp_vendor/patches-c1a477d3/0001-zenoh-e5db0ce-from-c1a477d3.patch
+++ b/zenoh_cpp_vendor/patches-c1a477d3/0001-zenoh-e5db0ce-from-c1a477d3.patch
@@ -1,0 +1,283 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index 45e757f9..e4c3042c 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -4226,7 +4226,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+  "arc-swap",
+@@ -4278,7 +4278,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-buffers"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "zenoh-collections",
+ ]
+@@ -4315,7 +4315,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-codec"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "tracing",
+  "uhlc",
+@@ -4327,7 +4327,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-collections"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+ ]
+@@ -4335,7 +4335,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-config"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "json5",
+  "nonempty-collections",
+@@ -4360,7 +4360,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-core"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "lazy_static",
+  "tokio",
+@@ -4371,7 +4371,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-crypto"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "aes",
+  "hmac",
+@@ -4384,7 +4384,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-ext"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "bincode",
+@@ -4403,7 +4403,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-keyexpr"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "getrandom 0.2.17",
+  "hashbrown 0.16.1",
+@@ -4418,7 +4418,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "zenoh-config",
+  "zenoh-link-commons",
+@@ -4438,7 +4438,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-commons"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4471,7 +4471,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-quic"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4496,7 +4496,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-serial"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "tokio",
+@@ -4514,7 +4514,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-tcp"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "socket2 0.5.10",
+@@ -4531,7 +4531,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-tls"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "base64",
+@@ -4560,7 +4560,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-udp"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "libc",
+@@ -4581,7 +4581,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-unixpipe"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "advisory-lock",
+  "async-trait",
+@@ -4603,7 +4603,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-unixsock_stream"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "nix 0.29.0",
+@@ -4621,7 +4621,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-vsock"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "libc",
+@@ -4639,7 +4639,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-link-ws"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "futures-util",
+@@ -4659,7 +4659,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-macros"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "proc-macro2",
+  "quote",
+@@ -4670,7 +4670,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-plugin-trait"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "git-version",
+  "libloading",
+@@ -4687,7 +4687,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-protocol"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "const_format",
+  "rand 0.8.5",
+@@ -4702,7 +4702,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-result"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "anyhow",
+ ]
+@@ -4710,7 +4710,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-runtime"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "lazy_static",
+  "ron",
+@@ -4724,7 +4724,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-shm"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "advisory-lock",
+  "async-trait",
+@@ -4753,7 +4753,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-stats"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "ahash",
+  "prometheus-client",
+@@ -4766,7 +4766,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-sync"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "arc-swap",
+  "event-listener",
+@@ -4780,7 +4780,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-task"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "futures",
+  "tokio",
+@@ -4793,7 +4793,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-transport"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "crossbeam-utils",
+@@ -4828,7 +4828,7 @@ dependencies = [
+ [[package]]
+ name = "zenoh-util"
+ version = "1.8.0"
+-source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#69b410e705e94531902a41a51b74d0c7b098264c"
++source = "git+https://github.com/eclipse-zenoh/zenoh.git?branch=main#e5db0ce8a52ddcd95c7449508434518db2f6fbec"
+ dependencies = [
+  "async-trait",
+  "const_format",

